TroopAnnouncements.html

```diff
@@ -65,15 +65,15 @@ <h2 class="w3-center">HTML slides</h2>
         
         <div class="w3-content" style="max-width:400px">
           <div class="mySlides w3-container w3-red">
-            <h1><b>Did You Know?</b></h1>
-            <h1><i>We plan to sell trips to the moon in the 2020s</i></h1>
+            <h1><b>Troop Outing</b></h1>
+            <h1><i>Our Next Troop Outing will be on Jan 13 2023, RSVP Via Scoutbook, or in person at next meeting.</i></h1>
           </div>
         
-          <img class="mySlides" src="https://media.discordapp.net/attachments/897388108591542304/1044398441981292634/b530938503bf4825a0a92b2f4be08296.png?width=393&height=1000" style="width:900%">
+          <img class="mySlides" src="https://imgur.com/qfDyUr0" style="width:900%">
         
           <div class="mySlides w3-container w3-xlarge w3-white w3-card-4">
-            <p><span class="w3-tag w3-red">New!</span>
-            <p>6 Crystal Glasses</p>
+            <p><span class="w3-tag w3-red">Merrit Bade University</span>
+            <p>We have been invited by  </p>
             <p>Only $99 !!!</p>
             <img class="myslides" src="https://media.discordapp.net/attachments/897388108591542304/1044397019013332992/SPOILER_7a4062d7f4b1410ebb66a8414d1590ee.png?width=1617&height=1002">
```

Comment:
> `<img class="myslides" src="https://media.discordapp.net/attachments/897388108591542304/1044397019013332992/SPOILER_7a4062d7f4b1410ebb66a8414d1590ee.png?width=1617&height=1002">`

I don't think this is the right image to use here.

Reply:
I agree. This was back when I was testing the slide thing and I apparently forgot to remove it. You may go ahead and remove it..half of the code for the troop announcement page is demo code from w3 schools..

Reply:
Image removed